### PR TITLE
Build and push docker images for Buffer API and Worker nodes

### PR DESCRIPTION
**Pull request by michaljurecko**
Part of: https://keboola.atlassian.net/browse/BA-33

-----

Secrets som nastavil:
image
image

---------

Cielom je pushnut Docker images.
K8S resources pre API a Worker node budu v dalsom PR.

-----

ENVs som fixol: https://github.com/keboola/kbc-stacks/pull/352

### CODE REVIEW

File: .github/workflows/release-service-buffer.yml [hunk 1]
```diff
@@ -3,18 +3,24 @@ name: 'Release: Buffer API'
...
-  ECR_PUSH_ROLE: "TODO"
-  ACR_REPOSITORY: "buffer-api"
   ACR_REGISTRY: "keboola.azurecr.io"
-  ACR_USERNAME: "buffer-api-push"
+
+  API_IMAGE_NAME: "keboola/buffer-api"
+  API_ECR_REPOSITORY: "keboola/buffer-api"
+  API_ECR_PUSH_ROLE: "arn:aws:iam::968984773589:role/kbc-ecr-BufferApiPushRole-8SI5TUN6TV99"
+  API_ACR_REPOSITORY: "buffer-api"
+  API_ACR_USERNAME: "buffer-api-push"
```
Comment (michaljurecko) [inline .github/workflows/release-service-buffer.yml:17]: Podla Templates API:
https://github.com/keboola/keboola-as-code/blob/ab14f2d77ac355b365a6f21d2ea30adb3292280b/.github/workflows/release-service-templates.yml#L9-L16

File: .github/workflows/release-service-templates.yml [hunk 2]
```diff
@@ -49,15 +49,19 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
 
+      - name: Docker login
+        uses: docker/login-action@v1
+        with:
+          username: ${{ env.DOCKERHUB_USER }}
+          password: ${{ env.DOCKERHUB_TOKEN }}
+
```
Comment (michaljurecko) [inline .github/workflows/release-service-templates.yml:57]: V release CLI mame aj docker login, pridal som ho aj sem.
Kedze v minulosti boli nejake problemy s rate-limitom, pre neprihlaseneho uzivatela.

File: .github/workflows/release-service-templates.yml [hunk 3]
```diff
@@ -49,15 +49,19 @@ jobs:
...
       - name: Build image
         uses: docker/build-push-action@v3
         with:
           load: true
           tags: ${{ env.IMAGE_NAME }}:${{ env.IMAGE_TAG }}
           context: .
           file: provisioning/templates-api/docker/Dockerfile
-          #cache-from: type=registry,ref=templates-api-build/cache:cache
-          #cache-to: type=registry,ref=templates-api-build/cache:cache,mode=max
```
Comment (michaljurecko) [inline .github/workflows/release-service-templates.yml:60]: Povodne sme tu chceli riesit aj cache, pre rychlejsi build, ale neriesime.

File: provisioning/buffer/local.sh [hunk 4]
```diff
@@ -13,6 +13,27 @@ if ! minikube status > /dev/null; then
...
+API_IMAGE="$BUFFER_API_REPOSITORY:$BUFFER_API_IMAGE_TAG"
+WORKER_IMAGE="$BUFFER_WORKER_REPOSITORY:$BUFFER_WORKER_IMAGE_TAG"
+echo
+echo "Building API image ..."
+echo "--------------------------"
+docker build -t "$API_IMAGE" -f "./docker/api/Dockerfile" "../../"
+echo
+echo "Building Worker image ..."
+echo "--------------------------"
+docker build -t "$WORKER_IMAGE" -f "./docker/worker/Dockerfile" "../../"
```
Comment (michaljurecko) [inline provisioning/buffer/local.sh:26]: Build images v CI.

**Comment by Halama** [comment]
Jdu mrknout.

**Review by Halama — APPROVED** [review]
Vypadá to ok, tak snad to projde po release.

**Pull request merged**